urdf/ur.urdf.xacro

```diff
@@ -57,6 +57,30 @@
    <!-- create link fixed to the "world" -->
    <link name="world" />
 
+   <link name="ground_plane">
```

Comment:
I think it would be nice to only add it when using gazebo, as otherwise this isn't necessarily required.